sktime/datatypes/_series/_check.py

```diff
@@ -546,6 +546,31 @@ def _index_equally_spaced(index):
 class SeriesXarray(ScitypeSeries):
     """Data type: xarray based specification of single time series.
 
+    Represents a single time series stored as an `xarray.DataArray`.
+    
+    Characteristics:
+    - Supports both univariate and multivariate series.
+    - Index (time) must be monotonic and equally spaced, if specified.
+    - Features (variables) must have unique names.
+
+    Example:
```

Comment:
for examples, you need to follow the numpydoc syntax, i.e., underline "Example", and use `>>> ` for code blocks. Have a look in other places of the code base.

I would not consider an example really needed here

Reply:
I've discarded the example block completely in the update